tools/test.py

```diff
@@ -1565,6 +1557,25 @@ def PrintCrashed(code):
   else:
     return "CRASHED (Signal: %d)" % -code
 
+IGNORED_SUITES = [
+  'addons',
+  'addons-napi',
+  'gc',
+  'internet',
+  'pummel',
+  'test-known-issues',
```

Comment:
it seemed reasonable to me that `test-known-issues` should not be part of the default test run, my main concern being that this could lead to inaccurate coverage results.